benchmarks/utils/argument_parsing.py

```diff
@@ -347,6 +338,15 @@ def __call__(self, parser, namespace, values, option_string=None):
         'sampled to participate again. This parameter is used in '
         '`BandedMatrixFactorizationMechanism`.')
 
+    argument_parser.add_argument(
+        '--noise_cohort_size',
+        type=int,
+        default=1000,
+        help=('The cohort size to use in calculating noise for DP. '
+              'If you run cohort_size=100 but noise_cohort_size=1000, '
+              'then your results will only be valid if running with '
```

Comment:
Please add description of what happens to standard deviation of noise when `noise_cohort_size` > `cohort_size`.